tests/installation/partitioning.pm

```diff
@@ -15,11 +15,24 @@ use strict;
 use warnings;
 use base "y2logsstep";
 use testapi;
-use version_utils qw(is_leap is_storage_ng);
+use version_utils qw(is_leap is_storage_ng is_sle sle_version_at_least);
 
 sub run {
     assert_screen 'partitioning-edit-proposal-button', 40;
 
+    # Add workaround since bsc#1089723 exists
```

Comment:
Do a `record_info` here

Reply:
As talked with Oliver offline, replace above code with: 
record_soft_failure('bsc#1089723 Make sure keep the existing windows partition');